roles/artifacts/tasks/edpm.yml

```diff
@@ -38,16 +38,15 @@
 
         - name: Extract Compute from zuul mapping if any
           when:
-            - crc_ci_bootstrap_networks_out is defined
```

Comment:
makes 100% sense, but the explanation is wrong: that parameter comes from the upstream job definition - and we don't have it for the VA path.

Care to update the RCA in the comment + commit message?

Reply:
Done